modules/openapi-generator/src/main/resources/typescript-angular/package.mustache

```diff
@@ -28,7 +28,12 @@
     "@angular/compiler": "^{{ngVersion}}",
     "core-js": "^2.4.0",
     "reflect-metadata": "^0.1.3",
-    "rxjs": "^{{rxjsVersion}}"
+    {{^useRxJS6}}
```

Comment:
why is this changed? it is not related to typescript-inversify

Reply:
I added a config's property`useRxJS6` to allow the user to decide the using of rxjs6 or rxjs5

Reply:
here you are modifying the `typescript-angular` file

Reply:
Sorry, I did a big mistake